ColabTool/LinkPost/index.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { PropTypes } from 'react';
+import {
+  Card,
+  Image,
+  LinkifiedText,
+  Text
+} from '@bufferapp/components';
+import style from './style.css';
+import Post from '../Shared/Post';
+
+// TODO: Padding between inner/outer card 0.5 for left, right, bottom?
+const LinkPost = ({
+  isConfirmingDelete,
+  isDeleting,
+  isWorking,
+  hovered,
+  imageSrc,
+  links,
+  linkAttachment,
+  manager,
+  onApproveClick,
+  onCancelConfirmClick,
+  onDeleteClick,
+  onDeleteConfirmClick,
+  onEditClick,
+  onMouseEnter,
+  onMouseLeave,
+  profile,
+  text,
+}) => {
+  const children = (
+    <div className={style['post-content']}>
+      <div className={style['post-content-text']}>
+        <LinkifiedText
+          links={links}
+        >
+          {text}
+        </LinkifiedText>
+      </div>
+      <div className={style['link-attachment-container']}>
+        <Card
```

Comment:
Heya @hharnisc - love this idea! Looks like we'll have to add some configs to the Link component - like `unstyled` to prevent the text from being underlined and such. 👍 😄